src/main/scala/com/amazon/deequ/analyzers/Histogram.scala

```diff
@@ -43,7 +44,9 @@ case class Histogram(
     binningUdf: Option[UserDefinedFunction] = None,
     maxDetailBins: Integer = Histogram.MaximumAllowedDetailBins,
     where: Option[String] = None,
-    computeFrequenciesAsRatio: Boolean = true)
+    computeFrequenciesAsRatio: Boolean = true,
+    aggregateFunction: String = Histogram.Count,
+    aggregateColumn: Option[String] = None)
```

Comment:
This is used when the aggregate function is sum. As a caller of this API, it is possible that this value is not provided when using `sum`. Is it possible to group this configuration into a single object? 

Check this parameter for reference: https://github.com/awslabs/deequ/commit/ef4c308a165d5928dc61e2349d907a232f8761f9#diff-4db9bbf900d2a35e16c36a158cbf3c4641d336e1930e6f343f2fef5773d0cea6R95